package.json

```diff
@@ -209,7 +209,7 @@
     "query-string": "5.1.1",
     "raf": "3.4.0",
     "raven-js": "3.27.0",
-    "rc-slider": "7.0.8",
+    "rc-slider": "^9.7.5",
```

Comment:
So we actually want to remove this dependency. There's no need for it.

This range component is a great example of something that should be in the design system. I'll port a dependency free version over to Palette then replace this.

Reply:
Reason being:

![](https://static.damonzucconi.com/_capture/3dPzKVxB.png)

This library somehow manages to include it's own tooltip engine; among other crazy dependencies.